zubhub_frontend/zubhub/src/views/edit_profile/EditProfile.jsx

```diff
@@ -364,26 +345,66 @@ function EditProfile(props) {
+                    >
+                      <InputLabel
+                        className={classes.customLabelStyle}
+                        htmlFor="password"
+                      >
+                        {t('editProfile.inputs.password.label')}
+                      </InputLabel>
+                      <OutlinedInput
+                        className={classes.customInputStyle}
+                        id="password"
+                        name="password"
+                        type={show_password ? 'text' : 'password'}
+                        onChange={props.handleChange}
+                        onBlur={props.handleBlur}
+                        endAdornment={
+                          <InputAdornment position="end">
+                            <IconButton
+                              aria-label="toggle password visibility"
+                              onClick={() =>
+                                handleSetState(handleClickShowPassword(state))
+                              }
+                              onMouseDown={handleMouseDownPassword}
+                              edge="end"
+                            >
+                              {show_password ? (
+                                <Visibility />
+                              ) : (
+                                <VisibilityOff />
+                              )}
+                            </IconButton>
+                          </InputAdornment>
+                        }
+                        labelWidth={calculateLabelWidth(
+                          t('editProfile.inputs.password.label'),
+                          document,
+                        )}
+                      />
                       <FormHelperText
                         className={classes.fieldHelperTextStyle}
                         error
                       >
-                        {props.status && props.status['init_phone'] && (
-                          <Typography
-                            color="textSecondary"
-                            variant="caption"
-                            component="span"
-                            className={classes.fieldHelperTextStyle}
-                          >
-                            {t('editProfile.inputs.phone.disabledHelperText')}
-                          </Typography>
-                        )}
-                        <br />
-                        {(props.status && props.status['phone']) ||
-                          (props.touched['phone'] &&
-                            props.errors['phone'] &&
+                        {(props.status && props.status['password']) ||
+                          (props.touched['password'] &&
+                            props.errors['password'] &&
                             t(
-                              `editProfile.inputs.phone.errors.${props.errors['phone']}`,
+                              `editProfile.inputs.password.errors.${props.errors['password']}`,
```

Comment:
I think the logic seems incorrect?
We need to show error if the status object has valid password property or the errors has valid password

Assuming status is the servers response

Reply:
@chaitanya-cloudknox  As far as I understand we are using the helper text over here either to show that user has not entered a password or the password length is too short similar to our sign-up page.
The response from the api whether password is valid and changes have been pushed or not is shown via a toast message (screenshots attached in the commit message)
Am I missing something?

Reply:
@chaitanya-cloudknox Yes, the issue for now is that we don't have a consistent way to show error, we are just showing the toast sometimes and sometimes we are just adding a small red text below the field. I feel this is fine for now.